src/Evolution/Systems/Cce/AnalyticSolutions/WorldtubeData.hpp

```diff
@@ -125,6 +125,8 @@ struct WorldtubeData : public PUP::able {
     return std::make_unique<Cce::InitializeJ::InverseCubic>();
   };
 
+  virtual bool noninertial_news() const noexcept { return false; }
```

Comment:
Could you indicate in the function name that it returns a boolean, e.g. with `use_` or `has_` or `should_output` or something like that? Else it reads like it returns a news scalar.